ssl/quic/quic_port.c

```diff
@@ -551,6 +552,7 @@ static QUIC_CHANNEL *port_make_channel(QUIC_PORT *port, SSL *tls, OSSL_QRX *qrx,
      * And finally init the channel struct
      */
     if (!ossl_quic_channel_init(ch)) {
+        OPENSSL_free(ch->qlog_title);
```

Comment:
This is actually a double-free. Any failure in ossl_quic_channel_init() results in `ch_cleanup` being called which frees `ch->qlog_title`. The only thing left to do is free `ch` itself.